src/pages/stats.tsx

```diff
@@ -268,3 +289,199 @@ async function biggestWin(matches: MatchWithPlayers[]) {
+        match.whiteEloChange,
+      )}
+    </span>
+  );
+};
+
+function matchhistoryDateToString(date: Date) {
+  const milisecondsBetween =
+    new Date(getDatePartFromDate(new Date())).getTime() -
+    new Date(getDatePartFromDate(date)).getTime();
+  const daysBetween = milisecondsBetween / (1000 * 60 * 60 * 24);
+  switch (daysBetween) {
+    case 0: {
+      return "Twoday:";
+    }
+    case 1: {
+      return "Yesterday:";
+    }
+    default: {
+      return (
+        date.toLocaleDateString("en-us", {
+          weekday: "long",
+          month: "long",
+          day: "numeric",
+        }) + ":"
+      );
+    }
+  }
+}
+
+function fancyInBetweenText(scoreDiff: number, losers: string) {
+  switch (true) {
+    case scoreDiff > 200:
+      return (
+        "cleaned the floor winning by " +
+        scoreDiff +
+        " points humiliating " +
+        losers +
+        " for life"
+      );
+    case scoreDiff > 180:
+      return "won by " + scoreDiff + " using their feet against" + losers;
+    case scoreDiff > 160:
+      return (
+        "needs to call an &#128511 ambulance &#128511 for " +
+        losers +
+        " as they lost by " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 140:
+      return "tryharded way too hard on " + losers + " winning by " + scoreDiff;
+    case scoreDiff > 120:
+      return (
+        "absolutely scooby doo doo'd " +
+        losers +
+        " by winning with " +
+        scoreDiff
+      );
+    case scoreDiff > 100:
+      return (
+        "found their inner Slater-power and smashed " +
+        losers +
+        " winnning by " +
+        scoreDiff
+      );
+    case scoreDiff > 80:
+      return (
+        "took a well deserved breather while winning against " +
+        losers +
+        " with " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 60:
+      return (
+        "comfortably manhandled " +
+        losers +
+        " winning with " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 50:
+      return (
+        "got an undeserved victory against " +
+        losers +
+        " winning by pathetic " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 40:
+      return (
+        "won a hard fought battle against " +
+        losers +
+        " with " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 30:
+      return (
+        "won by simply being better against " +
+        losers +
+        " winning by " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff >= 20:
+      return (
+        "won by sheer luck against " + losers + " with " + scoreDiff + " points"
+      );
+    case scoreDiff >= 5:
+      return (
+        "got the tightest of tightest wins against " +
+        losers +
+        " winning by " +
+        scoreDiff
+      );
+    default:
+      return "won ? against ";
+  }
+}
+
+function checkForFarming(
```

Comment:
If you took in both eloChanges you can figure out if it's a low elo change because of high elo players or not. Two 2100+ elo players doing a 1v1 would only get up to `16` for a win and then they would potentially get a :farmer: even though it was a fair match.

Essentially if the winning team gets little and the losing team loses little then it was a really unfair match.
```
Team 1: Avg 1750 elo
Team 2: Avg 1200 elo
Winner: Team 1
Team 1: gain 2
Team 2: loses 3
---
Team 1: avg 1610 elo
Team 2: avg 1356 elo
Winner: Team 1
Team 1: gains 11
Team 2: loses 12
```

Reply:
You can check if the `underdog` won by checking if the winning team gained more than the losing team lost. If winning team won more points than losing team lost it means they were in different k factor ranges so definite :facepalm:

Reply:
There's no difference in the absolute value of the winning and losing team right?
I adjusted the values after some testing. This seems like a simple solution which IMO just needs some sense of "did one of the players play against either of these opponents recently?" and take that into consideration when evaluating farming. For now I think this does the job.